master/reference/kube-controllers/configuration.md

```diff
@@ -80,6 +81,12 @@ To enable the node controller, perform the following two steps.
 
 This controller is only valid when using etcd as the {{site.prodname}} datastore.
 
+Additionally, the node controller can be configure to keeplabels on Kubernetes Node 
+objects and Calico Node objects in sync. By setting `SYNC_NODE_LABELS` to true (enabled 
+by default), labels on Kubernetes Node objects will act as the source of truth and
+will take precedence over labels on the corresponding Calico Node object if the values differ.
+However, labels on the Calico Node object that don't exist in the Kubernetes Node will remain as is.{: .alert .alert-info}
```

Comment:
IIRC, our style guide says "Node" should be lowercase since it's not a proper noun.